install.webapp

```diff
@@ -0,0 +1,48 @@
+---
+# This file is preprocessed by Jekyll to set the baseurl.
```

Comment:
Could you add a comment explaining the other .webapp file, and informing users that they should keep both files synchronized? Same thing in the other file.